Makefile

```diff
@@ -0,0 +1,16 @@
+BUILDER_OPTIONS = --force-clean --ccache --require-changes
```

Comment:
I'd remove it unless you're using it locally.

Reply:
I guess I was using it only because it was there and didn't think to remove it...will remove.